src/test/java/step2/LottoTest.java

```diff
@@ -0,0 +1,22 @@
+package step2;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import step2.domain.Lotto;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class LottoTest {
+
+    @Test
+    @DisplayName("두개 로또 번호 비교해서 몇개 일치하는지 테스트")
+    public void matchCountTest() {
+        Lotto lotto1 = new Lotto(new Integer[]{1, 2, 3, 4, 5, 6});
```

Comment:
```suggestion
        Lotto lotto1 = new Lotto("1, 2, 3, 4, 5, 6");
```

```suggestion
        Lotto lotto1 = new Lotto(1, 2, 3, 4, 5, 6);
```

위와 같이 다양한 인자를 전달해 Lotto를 생성하도록 지원한다면 테스트 코드 구현이나 Lotto를 사용하는 입장에서 좋지 않을까?